app/containers/DisplayWalletAccounts/DisplayWalletAccountsQrCodes/DisplayWalletAccountsQrCodes.jsx

```diff
@@ -99,6 +112,19 @@ class DisplayWalletAccountsQrCodes extends Component<Props, State> {
                   this.privateCanvas = node
                 }}
               />
+              <Button
```

Comment:
Let's actually remove this and only have the ability to copy encrypted and public. Having the private key QR on clipboard could introduce security concerns because unlike copying the text to the clipboard copying the QR in its current format using `withCopyCanvasToClipboard.js` requires us to write the file to disk

Reply:
Thanks for addressing this one 😄